src/components/Checkbox/Checkbox.tsx

```diff
@@ -136,7 +142,7 @@ export const Checkbox: React.FC<any> = chakra(
                 onChange,
               }
             : {
-                defaultIsChecked: false,
+                defaultChecked: false,
```

Comment:
Why was `Is` dropped from the prop name?

Reply:
I forgot to flag this, Chakra's new API has this as `defaultChecked`: https://chakra-ui.com/docs/components/checkbox#usage